packages/help-extension/src/index.tsx

```diff
@@ -341,7 +341,10 @@ function activate(
       );
       const copyright = (
         <span className="jp-About-copyright">
-          {trans.__('© 2015-%1 Project Jupyter Contributors', new Date().getFullYear())}
+          {trans.__(
+            '© 2015-%1 Project Jupyter Contributors',
+            new Date().getFullYear()
```

Comment:
Having a date there is actually saying something specific legally  - that there were significant changes in 2021, for example. I don't think we can automate this.

As a thought experiment, suppose we stopped making changes on jlab this year. The date would continue to update to 2022, 2023, etc., but it should not.

Since we are making a specific claim when we update this date, I think this should be manual.

Reply:
Thank for the explanation @jasongrout :) 

Sorry @jtpio :-|